src/actions/actionCreators.js

```diff
@@ -284,7 +283,6 @@ export const getProfile = () => dispatch => {
 
   Promise.all([getProfileInfo, getUserRecipes, getUserLikes, getForkedRecipesCount])
     .then(res => {
-      console.log(res.data);
```

Comment:
Is this function just there to log out all the data or does it call a function. If not I would delete the whole function.